lua/nvim-treesitter/locals.lua

```diff
@@ -127,17 +127,19 @@ end
 -- @param The full match path to append to
 -- @param The last match
 function M.recurse_local_nodes(local_def, accumulator, full_match, last_match)
-  if local_def.node then
-    accumulator(local_def, local_def.node, full_match, last_match)
-  else
-    for match_key, def in pairs(local_def) do
-      M.recurse_local_nodes(
-        def,
-        accumulator,
-        full_match and (full_match..'.'..match_key) or match_key,
-        match_key)
-    end
-  end
+	if(type(local_def) == "table") then
```

Comment:
to avoid indenting the whole function, you could just invert the condition and return immediately

Reply:
@madskjeldgaard you could follow this suggestions. But other wise it seems to be ready to merge.

Reply:
I've added a commit that fixes this ( I think) see below.